scheduler/ipp.c

```diff
@@ -5692,6 +5692,7 @@ create_local_printer(
   ippAddBoolean(con->response, IPP_TAG_PRINTER, "printer-is-accepting-jobs", (char)printer->accepting);
   ippAddInteger(con->response, IPP_TAG_PRINTER, IPP_TAG_ENUM, "printer-state", (int)printer->state);
   add_printer_state_reasons(con, printer);
+  printer->state_time = time(NULL);
```

Comment:
What's this change going to do?

Reply:
It prevents the temporary queue from removing in cupsdDeleteTemporaryPrinters() if someone asked for its creation recently.